### PR TITLE
Add note for unsupported Postgres SSL connection parameters

### PR DESCRIPTION
**Pull request by hashitop**
## Labels

- [X] inaccuracy
- [X] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout

### CODE REVIEW

**Review by justinretzolk — APPROVED** [review]
Looks good to me. Added @sudomateo to double check, as I'm unfamiliar with these parameters. Approving so I'm not the hold up.

File: content/source/docs/enterprise/before-installing/postgres-requirements.html.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ When providing optional extra keyword parameters for the database connection,
...
 
--> **Note:** See the PostgreSQL library documentation for more about [extra parameters related to sslmode](https://www.postgresql.org/docs/9.6/libpq-ssl.html). Terraform Enterprise provides a certificates file at `/tmp/cust-ca-certificates.crt`, which is required by the `verify-full` and `verify-ca` modes. Additional certificates can be added via the [CA Custom Bundle](../install/installer.html#certificate-authority-ca-bundle) setting.
+-> **Note:** See the PostgreSQL library documentation for more about [extra parameters related to sslmode](https://www.postgresql.org/docs/9.6/libpq-ssl.html). Terraform Enterprise provides a certificates file at `/tmp/cust-ca-certificates.crt`, which is required by the `verify-full` and `verify-ca` modes. Additional certificates can be added via the [CA Custom Bundle](../install/installer.html#certificate-authority-ca-bundle) setting, however, Terraform Enterprise currently does **not** support the client-side certificate parameters, `sslcert`, and `sslkey`.
```
Comment (justinretzolk) [inline content/source/docs/enterprise/before-installing/postgres-requirements.html.md:53]:
```suggestion
-> **Note:** See the PostgreSQL library documentation for more about [extra parameters related to sslmode](https://www.postgresql.org/docs/9.6/libpq-ssl.html). Terraform Enterprise provides a certificates file at `/tmp/cust-ca-certificates.crt`, which is required by the `verify-full` and `verify-ca` modes. Additional certificates can be added via the [CA Custom Bundle](../install/installer.html#certificate-authority-ca-bundle) setting, however, Terraform Enterprise currently does **not** support the client-side certificate parameters `sslcert` and `sslkey`.
```

**Review by justinretzolk — CHANGES_REQUESTED** [review]
On second though, these commas are unnecessary IMO.

**Comment by hashitop** [comment]
as per section 31.18.2, https://www.postgresql.org/docs/9.5/libpq-ssl.html

**Review by sudomateo — CHANGES_REQUESTED** [review]
Thank you for submitting this!

Instead of the current approach, I think you should create another note beneath the existing note specifying that verification of client certifications is not supported. Then link to section 
"32.18.2. Client Certificates" on [this page](https://www.postgresql.org/docs/9.6/libpq-ssl.html).

**Comment by hashitop** [comment]
@sudomateo is this ok?

image

**Comment by sudomateo** [comment]
@hashitop Looks good to me!

**Comment by hashitop** [comment]
Thank you!

**Review by hashitop — COMMENTED** [review]
Thank you!

**Comment by hashitop** [comment]
A few commits above due to the conflict in itself
I am going to squash it in now

**Pull request merged**